### PR TITLE
Add the tenant details page

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* Rebased with https://github.com/UKHomeOffice/right-to-rent-check/pull/4
Please merge above first before merging
* Use country selector
* Use the new date component
* Added styling taken from other hof apps
* Use latest hof-bootstrap

### CODE REVIEW

File: apps/right-to-rent-check/fields/index.js [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+  },
+  'tenant-country': {
+    mixin: 'select',
+    validate: 'required',
+    className: ['typeahead', 'js-hidden'],
+    options: [''].concat(require('homeoffice-countries').allCountries)
+  },
+  'tenant-dob': dateComponent('tenant-dob', {
+    validate: ['required', 'date', 'before']
+  }),
```
Comment (joefitter) [inline apps/right-to-rent-check/fields/index.js]: 🎉

File: apps/right-to-rent-check/translations/src/en/fields.json [hunk 2]
```diff
@@ -1,5 +1,33 @@
 {
-  "name": {
-    "label": "Name"
+  "living-status": {
```
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/fields.json]: where's the legend?

File: apps/right-to-rent-check/translations/src/en/fields.json [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+      "day": {
+        "label": "Day"
+      },
+      "month": {
+        "label": "Month"
+      },
+      "year": {
+        "label": "Year"
+      }
+    }
```
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/fields.json]: you don't need parts - these are the defaults. Only use parts if you want to use something other than Day, Month Year

File: apps/right-to-rent-check/translations/src/en/journey.json [hunk 4]
```diff
@@ -1,3 +1,3 @@
 {
-  "header": "My New HOF Service"
+  "header": "Home Office right to rent check"
```
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/journey.json]: could you check capitalisation here? Think this should be Title Case

File: apps/right-to-rent-check/translations/src/en/pages.json [hunk 5]
```diff
@@ -1,5 +1,17 @@
 {
-  "name": {
-    "header": "Personal details"
+  "request-property": {
+    "header": "Does the person already live in the property?"
```
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/pages.json]: Add this as the `legend` for the field, then remove the header. add legend: { className: 'visuallyhidden' } to the field config.

File: apps/right-to-rent-check/translations/src/en/journey.json (hunk 4 above)
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/journey.json]: Home Office Right to Rent Check

Comment (sulthan-ahmed) [inline apps/right-to-rent-check/translations/src/en/journey.json]: I wanted it the way you said but I asked Marion the content lead and she wanted as above

File: apps/right-to-rent-check/fields/index.js (hunk 1 above)
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/fields/index.js]: :man_dancing:

File: apps/right-to-rent-check/translations/src/en/fields.json (hunk 2 above)
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/translations/src/en/fields.json]: good spot

File: apps/right-to-rent-check/translations/src/en/fields.json (hunk 3 above)
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/translations/src/en/fields.json]: ahh cool, I did not know that.  I just followed the example daaaawwwwg 🐩  

I could add to the hof-bootstrap example one with parts and one without

Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/fields.json]: it was intentional in bootstrap, as you _may_ want to include custom ones, especially if including a different language. I don't think another example is _needed_ I don't want the example to be too convoluted, it should be in the readme and it _will_ be added to the bootstrap docs once we get round to them

File: apps/right-to-rent-check/translations/src/en/journey.json (hunk 4 above)
Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/journey.json]: fair

Comment (MarionDownie) [inline apps/right-to-rent-check/translations/src/en/journey.json]: I confirm this is the (non title) case!

Comment (joefitter) [inline apps/right-to-rent-check/translations/src/en/journey.json]: cheers @MarionDownie, just checking! 👍

**Comment by JoeChapman** [comment]
@joefitter are you happy to merge this PR now?

**Comment by joefitter** [comment]
approved but it won't let me merge for some reason. Another review needed? drone misconfigured?

**Comment by sulthan-ahmed** [comment]
Can you now see the merge button @joefitter ? I rebased with master and that seemed to fix the issue

**Comment by joefitter** [comment]
yep!

**Comment by sulthan-ahmed** [comment]
yayyy, thanks

**Pull request merged**